### PR TITLE
[GG-915] Add notifications styles

### PR DESCRIPTION
**Pull request by VilhelmNielsen**
Adds notification styles into theme.

JIRA: https://zendesk.atlassian.net/browse/GG-915

### CODE REVIEW

File: styles/_notifications.scss [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+.notification-notice {
+  background: #dbf3ff;
+  border-color: #b5e0f5;
+}
+
+.notification-notice .notification-icon::before {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='12' height='12' focusable='false' viewBox='0 0 12 12'%3E%3Cg fill='none' stroke='%23555555'%3E%3Cpath stroke-linecap='round' stroke-linejoin='round' d='M3.5 6l2 2L9 4.5'/%3E%3Ccircle cx='6' cy='6' r='5.5'/%3E%3C/g%3E%3C/svg%3E");
+}
+
+/* Alert / Lock */
```
Comment (luis-almeida) [inline styles/_notifications.scss]: I don't think we need need `notification-lock` as these will stay in HC as callouts.

File: styles/_notifications.scss [hunk 2]
```diff
@@ -0,0 +1,151 @@
+.notification {
```
Comment (luis-almeida) [inline styles/_notifications.scss]: Can we add a `/***** Notifications *****/` comment at the top of this partial so it segments this section in the compiled file?

Comment (VilhelmNielsen) [reply styles/_notifications.scss]: Amended

File: styles/_notifications.scss (hunk 1 above)
Comment (VilhelmNielsen) [reply styles/_notifications.scss]: I've removed it in a new commit 👍

**Pull request merged**